<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SACHINN" ignored="0" name="BankingApp Test in Chrome" tests="2" failures="1" timestamp="18 Feb 2020 09:55:11 GMT" time="24.371" errors="0">
  <testcase name="loginTest" time="12.992" classname="com.inetbanking.testCases.TC_LoginTest_001"/>
  <testcase name="loginTest" time="12.999" classname="com.inetbanking.testCases.TC_LoginTest_001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.aventstack.extentreports.ExtentObservable.lambda$saveTest$0(ExtentObservable.java:209)
at com.aventstack.extentreports.ExtentObservable$$Lambda$343/2078919047.accept(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at com.aventstack.extentreports.ExtentObservable.saveTest(ExtentObservable.java:209)
at com.aventstack.extentreports.ExtentReports.createTest(ExtentReports.java:234)
at com.aventstack.extentreports.ExtentReports.createTest(ExtentReports.java:247)
at com.inetbanking.utilities.Reporting.onTestSuccess(Reporting.java:51)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- BankingApp Test in Chrome -->
